src/Akka.Streams.Kafka.Tests/Logging/SimpleFileLoggerActor.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.IO;
+using Akka.Actor;
+using Akka.Event;
+
+namespace Akka.Streams.Kafka.Tests.Logging
+{
+    public class SimpleFileLoggerActor : ReceiveActor
```

Comment:
Is this just for debugging purposes?

Reply:
Yes. There is a `UseFileLogging` option in `KafkaIntegrationTests` class:

```csharp
/// <summary>
/// Allows to write logs to file (useful for debugging when tests are running forever and no output in console is available)
/// </summary>
private static readonly bool UseFileLogging = Environment.GetEnvironmentVariable("AKKA_STREAMS_KAFKA_FILE_LOGGING") != null;
```

So this logger is used if this option is set:
```csharp
if (UseFileLogging)
{
	config = config.WithFallback(ConfigurationFactory.ParseString("akka.loggers = [\"Akka.Streams.Kafka.Tests.Logging.SimpleFileLoggerActor, Akka.Streams.Kafka.Tests\"]"));
}
```